core/src/wallet/common/PersistentBlockchainDatabase.hpp

```diff
@@ -80,6 +80,15 @@ namespace ledger {
                     });
                 }
 
+                Future<Option<uint32_t>> getLastBlockHeight() override {
+                    return _persistentDB->GetLastBlock()

```

Comment:
Now we don't parse RawBlock, that is good, but we still read it.
Let keep this in mind for future.